### DISCUSSION TITLE
如何优雅的保持antd升级?

### DISCUSSION THREAD

**Discussion by luckyxutao** [ant-design/ant-design#35924] [General]
我们的项目是基于antd二次封装的，其中包含对一些组件样式覆盖(目前是锁定了版本)，请问有什么最佳实践可以同时满足个性化样式及antd升级呢？谢谢

**Comment by cwjTerrace** [comment]
- 关心antd的release note，一般api变更或者样式重构会列出来。
- 可以使用～锁定次版本号，不锁定修订版本号，尽量每个次版本号都及时更新，防止跳版本更新出现变更过多导致的问题。
- 使用Snapshot确保你的UI不会有意外的改变
